pages/dashboard/index.js

```diff
@@ -15,8 +11,8 @@ export default function Dashboard() {
 
   if (isLoading || !template) return <p>Loading...</p>;
   return (
-    <DashboardContainer>
+    <section>
```

Comment:
It is the same here.

Whatever you decide now to do with this, you can go on to the QA afterwards.

Reply:
Thanks Paul! 
I'm ready for Feierabend.